### PR TITLE
cgo cross-compile for freebsd

### PR DESCRIPTION
**Pull request by techknowlogick**
Provide pre-compiled cgo binaries for freebsd

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -780,6 +780,13 @@ ifeq ($(CI),true)
 	cp /build/* $(DIST)/binaries
 endif
 
+.PHONY: release-freebsd
+release-freebsd: | $(DIST_DIRS)
```
Comment (jolheiser) [inline Makefile:784]: I think this would need to be added to `release` as well, right?

File: Makefile [hunk 2]
```diff
@@ -780,6 +780,13 @@ ifeq ($(CI),true)
 	cp /build/* $(DIST)/binaries
 endif
 
+.PHONY: release-freebsd
```
Comment (delvh) [inline Makefile:783]:
```suggestion
.PHONY: release-freebsd-amd64
```

**Review by delvh — APPROVED** [review]
Well… Due to a lack of FreeBSD systems at hand I simply trust you that this compiles.

File: Makefile (hunk 1 above)
Comment (techknowlogick) [reply Makefile:784]: good catch

File: Makefile (hunk 2 above)
Comment (techknowlogick) [reply Makefile:783]: I'd prefer to keep as-is for when arm64 support for freebsd comes it'd be via this make target, just like how it was done with darwin.

**Review by jolheiser — COMMENTED** [review]
lgtm pending a successful build pipeline

**Comment by techknowlogick** [comment]
@jolheiser yup:
screenshot of gitea binary running on freebsd

**Review by jolheiser — APPROVED** [review]

**Pull request merged**